project/make.sh

```diff
@@ -58,6 +58,8 @@ DEFAULT_BUNDLES=(
 	cover
 	cross
 	tgz
+
+	fedora
```

Comment:
ya i was trying to keep the naming in the same fashion but should it be rpm and the ubuntu one we just have to keep as that :(